### DISCUSSION TITLE
New way to disable exstensions

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#223] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I found a new way to dissable extensions, may or may not work fo you, did for me. P.S need to be able to change your DNS
https://www.youtube.com/watch?v=qo-hx0tOYxI
credit to whoever made the tutorial, also credit to bypassi for making it

**Comment by ghost** [comment]
It works for every extension, yes yes

**Reply by ghost** [reply]
It works for all extensions, just see the bottom of the page. Don't self advertise btw

**Reply by The-Man-Is-Me** [reply]
Sorry, forgot to put proper credits, fixed now

**Reply by ghost** [reply]
@yeeteeyt sorry, I wont do it again :/

**Reply by ghost** [reply]
Also it doesn't work for securly, as you cannot open the page itself. It just says the person who set up this computer blocked it.

**Reply by The-Man-Is-Me** [reply]
that sucks, sorry

**Reply by ghost** [reply]
meh its fine, I can wait for the update, its not your fault

**Reply by ghost** [reply]
Oh, srry I'm dumb

**Reply by ghost** [reply]
all good :)

**Reply by GGboogie** [reply]
Does this work for Cisco Umbrella?

**Comment by frogmasamune** [comment]
For me it works until it displays the url you copy, when i go to the url it says chrome blocked it.

**Reply by ghost** [reply]
me too, its because you don't have goguardian (I have securly :/

**Reply by Theblackmamba824** [reply]
for sum reason this link shows up when i click on the link in the vid (https://marketingplatform.google.com/about/analytics/)

**Reply by Theblackmamba824** [reply]
pls help

**Reply by synapt0** [reply]
same

**Reply by frogmasamune** [reply]
you must not have the dns set correctly

**Reply by Theblackmamba824** [reply]
it is, I copied the dns in the description
